test_scripts/Polices/App_Permissions/ATF_HP_User_Consent_NO.lua

```diff
@@ -0,0 +1,309 @@
+                  {
+                    policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"
+                  })
+                local function to_run()
+                  self.hmiConnection:SendResponse(systemRequestId,"BasicCommunication.SystemRequest", "SUCCESS", {})
+                end
+                RUN_AFTER(to_run, 800)
+                self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+              end)
+          end)
+      end)
+  end
+
+end
+Precondition()
+
+--[[ Test ]]
+function Test:Step1_User_Disallow_New_Permissions_After_App_Activation()
+
+  commonFunctions:userPrint(34, "-- Test --")
+  local RequestIdActivateApp = self.hmiConnection:SendRequest("SDL.ActivateApp", {appID = self.applications["SPT"]})
+
+  EXPECT_HMIRESPONSE(RequestIdActivateApp,
+    { result = {
+        code = 0,
+        isPermissionsConsentNeeded = true,
+        isAppPermissionsRevoked = false,
+        isAppRevoked = false},
+      method = "SDL.ActivateApp"})
+
+  local RequestIdGetUserFriendlyMessage = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+  EXPECT_HMIRESPONSE(RequestIdGetUserFriendlyMessage,
+    { result = { code = 0,
+        messages = {{ messageCode = "DataConsent"}},
+        method = "SDL.GetUserFriendlyMessage"}})
+
+  local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", { appID = self.applications["SPT"] })
+  EXPECT_HMIRESPONSE(RequestIdListOfPermissions,
+    { result = {
+        code = 0,
+        allowedFunctions = {{name = "New_permissions"}} },
+      method = "SDL.GetListOfPermissions"})
+
+  :Do(function(_,data)
+      local functionalGroupID = data.result.allowedFunctions[1].id
+      self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent",
+        { appID = self.applications["SPT"], source = "GUI", consentedFunctions = {{name = "New_permissions", allowed = false, id = functionalGroupID} }})
+    end)
+  EXPECT_NOTIFICATION("OnPermissionsChange", {})
+  -- Permissions based on preloaded_pt file
+  --ToDo: (vikhrov) develop functions for extracting permissions for specific groups and compare it with permissionItem from OnPermissionsChange
+end
+
+function Test:Step2_Check_RPC_Disallowed_By_User()
+
+  local CorIdRAI = self.mobileSession:SendRPC("Show",
+    {
+      mediaClock = "00:00:01",
+      mainField1 = "Show1"
+    })
+  EXPECT_HMICALL("UI.Show", {}):Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, "UI.Show", "SUCCESS", { })
+    end)
+  EXPECT_RESPONSE(CorIdRAI, { success = true, resultCode = "SUCCESS"})
+end
+
+function Test:Step3_Check_Disallowed_RPC()
+  local cid = self.mobileSession:SendRPC("AddSubMenu",
+    {
+      menuID = 1000,
+      position = 500,
+      menuName ="SubMenupositive"
+    })
+  EXPECT_RESPONSE(cid, { success = false, resultCode = "DISALLOWED" })
+end
+
+--[[ Postcondition ]]
```

Comment:
Please organize section according to template https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32